open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/java/org/odpi/openmetadata/userinterface/uichassis/springboot/EgeriaUIPlatform.java

```diff
@@ -46,9 +45,10 @@ public static void main(String[] args) {
     public InitializingBean getInitialize()
     {
         return () -> {
-            if (!strictSSL)
-            {
-                HttpHelper.noStrictSSL();
+            // TODO: This section will be removed once the new implementation is complete and tested
+            if (!strictSSL) {
+                LOG.warn("strict.ssl is *DEPRECATED*! Please use egeria.ssl.client.verify system property or environment variable. If already set they will take precedence over strict.ssl. Note also that strict.ssl=true is the same as egeria.ssl.client.verify=false, and in the new implementation only affects client interaction");
```

Comment:
If you reach this point it is already set.
In the current implementation also affects only client interaction but on the JVM level which is more efficient

Reply:
yes as per above